src/js/containers/bulkDownload/MetadataDownloadContainer.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * MetadataDownloadContainer.jsx
```

Comment:
I think we could combine the container and component into a single component, since there's no API request/connecting to redux logic in this file.

Reply:
Agreed, moved everything into a single component `MetadataDownload`